documentation/configuring_bolt.md

```diff
@@ -1,8 +1,41 @@
 # Configuring Bolt
 
-You can configure Bolt's options and features at a project level, a user level,
-or a system-wide level. Unless your use case requires setting user-specific or
-system-wide configurations, configure Bolt at the project level. 
+Bolt has many options and features that can be configured to suit your
+project's needs. In general, Bolt configuration falls into four categories:
+
+- **Bolt behavior**
+
+  Configure how Bolt itself runs, such as the format to use when displaying
+  output or how many threads to use when connecting to targets.
```

Comment:
```suggestion
- **Bolt behavior:** Configure how Bolt itself runs, such as the format to use when 
  displaying output or how many threads to use when connecting to targets.
```

Reply:
Can we collapse the rest of the bulleted list like this? Mostly a style thing, but I think it'll look better in the output too.